runtime/vm/j9vm.tdf

```diff
@@ -786,7 +786,7 @@ TraceEvent=Trc_VM_classInitStateMachine_verifyFlattenableField Group=classinit O
 TraceEvent=Trc_VM_classInitStateMachine_prepareFlattenableField Group=classinit Overhead=1 Level=3 Template="prepare flattenable field clazz=%p"
 TraceEvent=Trc_VM_classInitStateMachine_initFlattenableField Group=classinit Overhead=1 Level=3 Template="initialize flattenable field clazz=%p"
 
-TraceEntry=Trc_VM_GetCompleteNPEMessage_Entry Overhead=1 Level=3 Template="GetCompleteNPEMessage - bcCurrentPtr (0x%p) bcCurrent (0x%x) romClass (0x%p) npeCauseMsg (%s)"
+TraceEntry=Trc_VM_GetCompleteNPEMessage_Entry Obsolete Overhead=1 Level=3 Template="GetCompleteNPEMessage - bcCurrentPtr (0x%p) bcCurrent (0x%x) romClass (0x%p) npeCauseMsg (%s)"
```

Comment:
Is this legal (obsoleting and reusing the same tracepoint later)?

Reply:
No, it isn't.
Changed the new name to `Trc_VM_GetCompleteNPEMessage_Entry2`.

Reply:
Given that these tracepoints have never been used (preview feature), it might be best to simply change the format string in place.

Reply:
Agreed, modified initial trace point `Trc_VM_GetCompleteNPEMessage_Entry` instead.